bin/regenerate-offline

```diff
@@ -6,14 +6,21 @@ GOVUK_FRONTEND_PREFIX=https://github.com/alphagov/govuk-frontend/releases/downlo
 GOVUK_FRONTEND_ARCHIVE_NAME=release-v${RELEASE}.zip
 GOVUK_FRONTEND_URL="${GOVUK_FRONTEND_PREFIX}${GOVUK_FRONTEND_ARCHIVE_NAME}"
 
-wget ${GOVUK_FRONTEND_URL}
+# Test file exists and is a regular file
+if [[ ! -f ${GOVUK_FRONTEND_ARCHIVE_NAME} ]]; then
+  # we use wget to support the redirect to the underlying files being served from github-releases.githubusercontent.com
+  wget -O ${GOVUK_FRONTEND_ARCHIVE_NAME} ${GOVUK_FRONTEND_URL}
+fi
 if [[ ! -f "${GOVUK_FRONTEND_ARCHIVE_NAME}" ]]; then
   echo "${GOVUK_FRONTEND_ARCHIVE_NAME} does not exist."
   exit 1
 else
-  unzip ${GOVUK_FRONTEND_ARCHIVE_NAME} -d ./offline
+  # Unzip, but Never overrite existing files
```

Comment:
Why not overwrite?

Reply:
It was an idempotence thing, but given that:
- if running in a CI workflow, it will only run once
- if running locally, we might want to be prompted to overwrite files in the /offline folder
I've removed the `-n` flag